src/components/settings-about.tsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+
+import {ScrollView, View, useWindowDimensions} from 'react-native';
+import {useSafeAreaInsets} from 'react-native-safe-area-context';
+
+import {Color} from '@app/colors';
+import {Icon, IconButton, LottieWrap, Spacer, Text} from '@app/components/ui';
+import {createTheme} from '@app/helpers';
+import {I18N} from '@app/i18n';
+
+type SettingsAboutProps = {
+  onPressSite: () => void;
+  onPressDoc: () => void;
+  onPressDiscord: () => void;
+};
+
+export const SettingsAbout = ({
+  onPressSite,
+  onPressDoc,
+  onPressDiscord,
+}: SettingsAboutProps) => {
+  const insets = useSafeAreaInsets();
+  const windowWidth = useWindowDimensions().width;
+  return (
+    <ScrollView
+      style={{paddingBottom: insets.bottom}}
+      contentContainerStyle={styles.content}>
+      <Spacer style={styles.animation}>
+        <LottieWrap
+          style={[
+            styles.imageStyle,
+            {width: windowWidth, height: windowWidth * 0.9},
+          ]}
+          source={require('../../assets/animations/first-screen-animation.json')}
+          autoPlay
+          loop
+        />
+      </Spacer>
+      <Text t14 i18n={I18N.settingsAboutTitle} style={styles.title} />
+      <View style={styles.buttons}>
+        {/* <IconButton onPress={onPressRate} style={page.button}>
+          <StarIcon color={GRAPHIC_BASE_1} />
+          <Text t11 style={page.buttonText}>
+            Rate ISLM wallet App
+          </Text>
+          <Spacer />
+           <Icon i24 name="arrow_forward" color={Color.graphicSecond3} />
+        </IconButton> */}
+        <IconButton onPress={onPressSite} style={styles.button}>
+          <Icon i24 name="global" color={Color.graphicBase1} />
+          <Text t11 i18n={I18N.settingsAboutVisit} style={styles.buttonText} />
+          <Spacer />
+          <Icon i24 name="arrow_forward" color={Color.graphicSecond3} />
+        </IconButton>
```

Comment:
I think we can refactor these and move it in a separate component which accepts icon name, name and url

Reply:
done